tests/test_asn1_codec.py

```diff
@@ -0,0 +1,136 @@
+import tuf
+import tuf.util
+import tuf.asn1_codec as asn1_codec
+import tuf.conf
+import unittest
+
+tuf.conf.METADATA_FORMAT = 'json'
+
+class TestASN1Conversion(unittest.TestCase):
+
+
+  # THIS NEXT TEST fails because the TUF root.json test file in question here
+  # uses an RSA key, which the ASN1 conversion does not yet support.
+  # TODO: <~> FIX.
+  # Our ASN1 conversion doesn't seem to support RSA keys. In particular, it is
+  # being assumed that the key values are hex strings ('f9ac1325...') but an
+  # RSA public key value is e.g. '-----BEGIN PUBLIC
+  # KEY------\nMIIBojANBgk...\n...'
+  def test_1_root_partial_convert(self):
```

Comment:
I just pulled in your latest commits, so am now on commit 892c0bc333727ae6287d998ca7ddad0f69324308

I still get the PyAsn1Error exception.